src/components/Calculator.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+function Calculator() {
```

Comment:
- [ ] From the requirements of this project, this should be a class-based component. Please let's fix it and get closer to approval